### PR TITLE
Add the home page sources

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the sources of the home page blades.

### CODE REVIEW

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/character.bal [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/service-designing/graphql-designer/starwars/character.bal:1]:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/modules/datasource/datasource.bal [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/service-designing/graphql-designer/starwars/modules/datasource/datasource.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/droid.bal [hunk 3]
```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/droid.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/datasource.bal [hunk 4]
```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/datasource.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/types.bal [hunk 5]
```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/types.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/utils.bal [hunk 6]
```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/utils.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/service.bal [hunk 7]
```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/service.bal:1]: Update license

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/starship.bal [hunk 8]
```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (gigara) [inline en/docs/home-page-sources/try-services/graphql-try-it/starwars/starship.bal:1]: Update license

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/droid.bal [hunk 9]
```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/droid.bal:1]: Updated.

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/human.bal [hunk 10]
```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/human.bal]: Fixed.

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/modules/datasource/datasource.bal (hunk 2 above)
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/modules/datasource/datasource.bal]: Fixed.

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/service.bal [hunk 11]
```diff
@@ -0,0 +1,157 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/service.bal]: Fixed.

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/tests/tests.bal [hunk 12]
```diff
@@ -0,0 +1,318 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/tests/tests.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/character.bal [hunk 13]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/character.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/droid.bal (hunk 3 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/droid.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/datasource.bal (hunk 4 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/datasource.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/utils.bal (hunk 6 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/utils.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/service.bal (hunk 7 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/service.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/starship.bal (hunk 8 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/starship.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/tests/tests.bal [hunk 14]
```diff
@@ -0,0 +1,318 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/tests/tests.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/tests/utils.bal [hunk 15]
```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/tests/utils.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/types.bal (hunk 5 above)
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/modules/datasource/types.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/human.bal [hunk 16]
```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/human.bal]: Fixed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/target/build [hunk 17]
```diff
@@ -0,0 +1,8 @@
+{
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/target/build]: Removed.

File: en/docs/home-page-sources/service-designing/graphql-designer/starwars/target/build [hunk 18]
```diff
@@ -0,0 +1,8 @@
+{
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/service-designing/graphql-designer/starwars/target/build]: Removed.

File: en/docs/home-page-sources/try-services/graphql-try-it/starwars/README.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
+# Starwars GraphQL API
+
+[![Star on Github](https://img.shields.io/badge/-Star%20on%20Github-blue?style=social&logo=github)](https://github.com/ballerina-platform/module-ballerina-graphql)
+
+_Authors_: @DimuthuMadushan  
+_Reviewers_: @shafreenAnfar @ThisaruGuruge
```
[the remark this answers is not in the document]
Comment (praneesha) [reply en/docs/home-page-sources/try-services/graphql-try-it/starwars/README.md]: Removed.

**Review by gigara — APPROVED** [review]

**Pull request merged**